From 4389421ec1414fc4485fefa043d8c72e1e2228b2 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=E9=82=B1=E5=BB=BA=E6=96=8C?= <qjb@rock-chips.com>
Date: Sat, 20 Oct 2012 15:08:57 +0800
Subject: [PATCH] i2s: Remove compiler warnings

   sound/soc/rk29/rk29_i2s.h:247:6: warning: "CONFIG_SND_SOC_RT5631" is not defined
   sound/soc/rk29/rk30_i2s.c:200:6: warning: "CONFIG_SND_SOC_RT5631" is not defined
---
 sound/soc/rk29/rk29_i2s.h | 2 +-
 sound/soc/rk29/rk30_i2s.c | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/sound/soc/rk29/rk29_i2s.h b/sound/soc/rk29/rk29_i2s.h
index 374c773f65b0..116001a99860 100755
--- a/sound/soc/rk29/rk29_i2s.h
+++ b/sound/soc/rk29/rk29_i2s.h
@@ -244,7 +244,7 @@ extern struct snd_soc_dai_driver rk29_i2s_dai[];
 extern struct snd_soc_dai rk29_i2s_dai[];
 #endif
 
-#if (CONFIG_SND_SOC_RT5631)
+#ifdef CONFIG_SND_SOC_RT5631
 extern struct delayed_work rt5631_delay_cap; //bard 7-16
 #endif
 
diff --git a/sound/soc/rk29/rk30_i2s.c b/sound/soc/rk29/rk30_i2s.c
index f4ee6770f381..9a2517de26e6 100755
--- a/sound/soc/rk29/rk30_i2s.c
+++ b/sound/soc/rk29/rk30_i2s.c
@@ -197,7 +197,7 @@ static void rockchip_snd_rxctrl(struct rk29_i2s_info *i2s, int on, bool stopI2S)
 		}
 
 	  flag_i2s_rx = 1;
-#if (CONFIG_SND_SOC_RT5631)
+#ifdef CONFIG_SND_SOC_RT5631
 //bard 7-16 s
 		schedule_delayed_work(&rt5631_delay_cap,HZ/4);
 //bard 7-16 e
-- 
2.35.3

